### DISCUSSION TITLE
Traduction

### DISCUSSION THREAD

**Discussion by Chabi01** [chamilo/chamilo-lms#5359] [General]
Bonjour,
J'utilise la toute dernière version de Chamilo.
J'aimerai changer quelques traductions mais je n'arrive pas à m'en sortir...
Je ne trouve pas comment on fait des "sous langage" (trouvé sur le net mais introuvable sur mon Chamilo) et j'ai également cherché dans le fichier trac4all.inc.php en français, mais je ne trouve pas la chaine de texte...
Mon objectif : quand on n'est pas connecté à la plateforme, on peut voir dans la barre de menu "Cours". Je voudrais changer ce mot en "Liste des formations".

Merci de votre aide :)
Bien cordialement,
Xavier

**Comment by LudiscapeXApi** [comment]
Bonjour,

Une solution rapide : 
Modifier les 3 termes suivants dans le fichier trad4all.inc.php: 
`
$AdminCourses = "Cours";
$Courses = "Cours";
$Course = "Cours";
`

https://github.com/chamilo/chamilo-lms/blob/1.11.x/main/lang/french/trad4all.inc.php

**Comment by Chabi01** [comment]
Merci :)
En fait, j'ai complètement repris le fichier trad4all.inc.php pour avoir une plateforme orientée "formations" à la place de "cours" et "Formateurs" à la place de "professeur".
Cela m'a pris un peu de temps, mais cela fonctionne :)
Encore merci !
Xavier
